client/ios/DivKit/Actions/DivActionHandler.swift

```diff
@@ -187,11 +187,13 @@ public final class DivActionHandler {
     )
 
     if !isDivActionURLHandled {
+      let referer = action.resolveReferer(expressionResolver)
+      logger.log(url: url, referer: referer, payload: action.payload)
```

Comment:
Seems like a breaking change. All non div-action urls will be sent to DefaultDivActionLogger (which actually makes http-requests). I don't think it is a good idea.